celavi/pylca_celavi/insitu_emission.py

```diff
@@ -198,15 +157,15 @@ def runner(tech_matrix,F,yr,i,j,k,final_demand_scaler,process,df_with_all_other_
 
     
     res = pd.DataFrame()
-    res= solver_optimization(tech_matrix, F,process,df_with_all_other_flows)
+    res = solver(tech_matrix, F, process, df_with_all_other_flows) 
     res['value'] = res['value']*final_demand_scaler
     if  not res.empty:        
        res.loc[:,'year'] =  yr
        res.loc[:,'facility_id'] = i
        res.loc[:,'stage'] = j
        res.loc[:,'material'] = k
     else:        
-       print(f"Insitu emission optimization failed for {k} at {j} in {yr}")
+       print(f"Insitu emission calculation failed for {k} at {j} in {yr}")
     
     res = electricity_corrector_before20(res)
     res['route_id'] = None
```

Comment:
The `route_id` parameter can be removed from the function definition and the docstring

Reply:
Removed